src/Jalalian.php

```diff
@@ -354,6 +354,11 @@ public function subDays(int $days = 1): Jalalian
         return static::fromCarbon($this->toCarbon()->subDays($days));
     }
 
+    public function subDay(): Jalalian
+    {
+        return static::fromCarbon($this->toCarbon()->subDays(1));
```

Comment:
use `subDays(1)` here

Reply:
I see, I wanted it to be like Carbon and support the same name, but I can change it.

Reply:
Done.